src/pages/careers/locations/_location-layout.js

```diff
@@ -15,6 +15,16 @@ const Pin = styled.img`
     height: 24px;
     margin-right: 13px;
 `
+
+const StyledBackground = styled(BackgroundImage)`
+    min-width: 100%;
+    height: 80rem;
+    object-fit: contain;
+    @media ${device.tabletL} {
+        height: 65.3rem;
+    }
+`
+
 const StyledContainer = styled(Container)`
```

Comment:
We should use `styled(Flex)` instead of `Container` since we are setting `flex-direction` property

Reply:
If the intention is to use `Container` not `Flex` you can test and remove the unrelated properties 😅

Reply:
Apparently this `Container` is not the `Container` we suspected. It is a custom `CareerContainer` component in `/career/_layout-components/_container.js` which is not based off the `Flex` Component :laughing: 

To avoid any confusion I renamed all instances of Container in this file to `CareerContainer`

Reply:
I see 😂 we can move forward with it then 👍🏻